walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,174 @@
 
 package com.google.sps;
 
+import java.util.List;
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.LinkedList;
+import com.google.sps.TimeRange;
 
+/**
+ * This class exposes the {@code query} method, which finds available time slots
+ * for a meeting with a list of mandatory and optional attendees.
+ */
 public final class FindMeetingQuery {
```

Comment:
Since this interface is exposed, you might want to check if the inputs you get are valid.